src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import classNames from 'classnames';
+import copyToClipboard from 'copy-to-clipboard';
+
+import Button from '~core/Button';
+import { FormSection } from '~core/Fields';
+import Tag from '~core/Tag';
+import Icon from '~core/Icon';
+import { Tooltip } from '~core/Popover';
+
+import { Motion, MotionStatus } from '../../constants';
+
+import styles from './StreamingStagesLocked.css';
+
+const MSG = defineMessages({
+  paidToDate: {
+    id: `dashboard.ExpenditurePage.Stages.StreamingStage.StreamingStagesLocked.paidToDate`,
```

Comment:
You need to add an "s" to all ids for StreamingStage to become **StreamingStages**